### PR TITLE
GKEStartPodOperatorAsync

### PR DESCRIPTION
**Pull request by raphaelauv**
close  : https://github.com/astronomer/astronomer-providers/issues/177

### CODE REVIEW

**Review by kaxil — COMMENTED** [review]
Thank you very much for starting this PR ❤️ , do let us know if you would like any help getting it across the line

File: setup.cfg [hunk 1]
```diff
@@ -59,12 +59,7 @@ cncf.kubernetes =
 databricks =
     apache-airflow-providers-databricks>=2.2.0
 google =
-    apache-airflow-providers-google
-    # This is needed currently to install without constraints
-    # otherwise pip will loop around to find correct version that satisfies all requirements
-    # and will never install the correct version
-    # Related to https://github.com/googleapis/google-cloud-python/issues/10566
-    google-api-core>=1.25.1,<2.0.0
+    apache-airflow-providers-google==7.0.0
```
Comment (kaxil) [inline setup.cfg:62]:
```suggestion
    apache-airflow-providers-google>=7.0.0
```

File: setup.cfg [hunk 2]
```diff
@@ -122,14 +117,13 @@ all =
     apache-airflow-providers-amazon>=3.0.0
     apache-airflow-providers-cncf-kubernetes>=3
     apache-airflow-providers-databricks>=2.2.0
-    apache-airflow-providers-google
+    apache-airflow-providers-google==7.0.0
```
Comment (kaxil) [inline setup.cfg:120]:
```suggestion
    apache-airflow-providers-google>=7.0.0
```

**Review by kaxil — COMMENTED** [review]
Can you add an example DAG and tests similar to https://github.com/astronomer/astronomer-providers/blob/main/tests/cncf/kubernetes/operators/test_kubernetes_pod.py please.

Do let us know if you want to take it from here

**Comment by pankajastro** [comment]
In airflow repo, the sync version of this operator has file name`kubernetes_engine.py` what about keeping the same name here?

**Comment by pankajastro** [comment]
Can you please add the relevent description for this PR?

**Review by bharanidharan14 — COMMENTED** [review]
Looks fine to me can you address one more comment on the example DAG

File: astronomer/providers/google/cloud/operators/kubernetes_engine.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+from airflow.utils.context import Context
+
+from astronomer.providers.cncf.kubernetes.operators.kubernetes_pod import (
+    KubernetesPodOperatorAsync,
+)
+
+
+class GKEStartPodOperatorAsync(KubernetesPodOperatorAsync):
+    """
+    Executes a task in a Kubernetes pod in the specified Google Kubernetes
```
Comment (phanikumv) [inline astronomer/providers/google/cloud/operators/kubernetes_engine.py:17]:
```suggestion
    Executes a task aynchronously in a Kubernetes pod in the specified Google Kubernetes
```

Comment (raphaelauv) [reply astronomer/providers/google/cloud/operators/kubernetes_engine.py]: thanks

**Review by pankajastro — APPROVED** [review]

**Review by phanikumv — APPROVED** [review]
Congrats @raphaelauv on your first merged PR

**Comment by pankajastro** [comment]
Hi @raphaelauv How did you test it? I'm trying to run it but it is failing, tracking my investigation here https://github.com/astronomer/astronomer-providers/issues/390

**Pull request merged**